xlive/Blam/Engine/interface/hud.cpp

```diff
@@ -90,6 +43,11 @@ void set_crosshair_and_text_size(float size)
 	*Memory::GetAddress<float*>(0x464028, 0x0) = original_crosshair_text_scale * size * (1 / k_hud_upscale_size);
 }
 
+float* get_crosshair_and_text_size()
```

Comment:
`get_crisshair_and_text_size` ? pretty sure this is hud scaling.

Reply:
Well yeah, it scales some elements of the hud, I'm not necessarily talking about what it does exactly but what it actually represents, in this case 'hud_scale'.
Can't check rn but the other offset is probably the `hud_size`, which is practically `hud_scale` (by res) multiplied by `size` = `hud_size`. hud size scaled by res to be more specific.

Reply:
I used scale and size interchangeably when naming the globals and functions
It would probably be better to have the crosshair_and_text_size named secondary_hud_scale and the regular hud_scale called primary_hud_scale.

Unless you've got a better name since both scales control different parts of the hud.

Reply:
That would be better.

Reply:
changed the globals and the function names